ext/session/session.c

```diff
@@ -485,19 +485,24 @@ PHPAPI int php_session_valid_key(const char *key) /* {{{ */
 /* }}} */
 
 
-static void php_session_gc(void) /* {{{ */
+static zend_long php_session_gc(zend_bool immediate) /* {{{ */
 {
```

Comment:
in that case,  why you don't  use zend_long num?  I don't care about zend_long or int, just seems a little ugly you are doing casting without a good reason.